tests/Usages/Reproducer2Test.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types = 1);
+
+namespace Spaze\PHPStan\Rules\Disallowed\Usages;
+
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+
+class Reproducer2Test extends RuleTestCase
```

Comment:
If I read this test correctly it is now independent from the disallowed-calls extension package, right?

if so, I think we should open a phpstan-src issue and add the reproducer there

Reply:
I will port it over into the fixing PR at https://github.com/phpstan/phpstan-src/pull/3944

Reply:
Perfect, and yes, it is independent now.